### PR TITLE
completed POST topics

### PR DESCRIPTION
**Pull request by Seyaji**
completed the code for POST topics and its test

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -31,18 +31,35 @@ describe('API tests', () => {
...
-      describe('status 404: tests API paths for 400 error handling', () => {
-         const badPath = (path) => path.replace(/[a-z]$/i, 'badPath')
+      // Utility Test Functions
+      const badPath = (path) => path.replace(/[a-z]$/i, 'badPath')
+      
+      describe('status 404: tests GET API paths for 400 error handling', () => {
          const APIPaths = [
             [badPath('/api/topics'), `path not found...`]
          ]
          test.concurrent.each(APIPaths)('.get(%s)', (path, expected) => {
```
Comment (popatre) [inline __tests__/app.test.js:58]: I dont know if its somewhere else in the file, but not sure what 'expected' is expected to be. 

i think Vel suggested you dont use the concurrent thing.

File: __tests__/app.test.js [hunk 2]
```diff
@@ -31,18 +31,35 @@ describe('API tests', () => {
...
+         test('status 201: responds with newly added park to the database', () => {
+            const newTopic = { slug: 'coding', description: 'Javascript 101!' }
+            return request(app)
+            .post('/api/topics')
+            .send(newTopic)
+            .expect(201)
+            .then(({ body }) => {
+               expect(body.topics).toEqual({
+                  slug: 'coding',
+                  description: 'Javascript 101!'
```
Comment (popatre) [inline __tests__/app.test.js:44]: You could also just add another expect to check the length of the topics after. It should be one more added to the length

**Review by popatre — CHANGES_REQUESTED** [review]

File: __tests__/app.test.js (hunk 2 above)
Comment (Seyaji) [reply __tests__/app.test.js:44]: @popatre how would I go about doing this in the same test without making another request to the API? 
can you provide some guidance on this tomorrow, thanks :)

**Review by Augs0 — APPROVED** [review]
Looks good but consider adding some error handling in this endpoint e.g. what happens if I don't send a slug or description or send another key?

**Pull request merged**